### ISSUE TITLE
date format change

### ISSUE THREAD

**Issue by KAYABLE** [hammy1/BDStuff#25]
is there a way i can change the date from (mm/dd/yyyy) to (dd/mm/yyyy)

**Comment by ezeholz** [comment]
Yes! There is a way. You can go into dateViewer settings, and unselect the following option
![image](https://user-images.githubusercontent.com/30635862/156009296-1962df88-b1d9-416a-8e77-f6bb1338f24f.png)

**Comment by ezeholz** [comment]
You should also uncheck this option
![image](https://user-images.githubusercontent.com/30635862/156011152-65d7ccc5-d518-450c-bef8-08734a162877.png)
